### PR TITLE
add patch for media revisions ui issue 

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Opened issue for feature request and developed patch. Merge request opened.
https://www.drupal.org/project/media_revisions_ui/issues/3247661

Added patch to local files.

## Testing
Check out branch and run composer install. Ensure patch applies.
Make an edit to a media item locally (make sure to create new revision) and then view revisions tab.

## Metadata

| Question | Answer |
|----------|--------|

| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | https://www.drupal.org/project/media_revisions_ui/issues/3247661
https://thinkoomph.jira.com/browse/RIGA-158

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -122,6 +122,9 @@
         "patches": {
             "drupal/core": {
                 "2259567 - Unnecessary restrictions on logo format: Can't upload replacement SVG logo": "patches/drupal-core-2259567-allow-svg.patch"
+            },
+            "drupal/media_revisions_ui": {
+                "3247661 - Add fielname": "patches/3247661-media-revisions-filename.patch"
```
Comment (jordan-caldwell) [inline composer.json:127]: typo

File: patches/3247661-media-revisions-filename.patch [hunk 2]
```diff
@@ -0,0 +1,55 @@
+diff --git a/src/Controller/MediaRevisionController.php b/src/Controller/MediaRevisionController.php
+index e291dad8e80e8dc2c3e0b59e170d30c424d8b507..bd4b4beaa0c11d77f116a5eb9168d42e3e3fd753 100644
+--- a/src/Controller/MediaRevisionController.php
++++ b/src/Controller/MediaRevisionController.php
+@@ -82,6 +82,7 @@ class MediaRevisionController extends ControllerBase {
+     $langname = $media->language()->getName();
+     $languages = $media->getTranslationLanguages();
+     $hasTranslations = (count($languages) > 1);
++    $hasFileField = ($media->hasField('field_file'));
```
Comment (jordan-caldwell) [inline patches/3247661-media-revisions-filename.patch:9]: Are these extra parenthesis required?

**Review by jordan-caldwell — CHANGES_REQUESTED** [review]
Looks good. Just one typo and a question

File: patches/3247661-media-revisions-filename.patch (hunk 2 above)
Comment (ben-hamelin) [reply patches/3247661-media-revisions-filename.patch]: nope, we can do away with them!

**Review by jordan-caldwell — APPROVED** [review]
Looks good!

**Pull request merged**